tensorbay/opendataset/COCO2017/catalog.json

```diff
@@ -0,0 +1,600 @@
+{
+    "BOX2D": {
+        "categories": [
```

Comment:
Sorting `categories` according to alphabetic ordering is better?

Reply:
This is the order given by the official, and it can also be consistent with the order of the categories in panoptic mask.